frontend/public/components/machine-set.tsx

```diff
@@ -238,7 +239,7 @@ const MachineSetDetails: React.FC<MachineSetDetailsProps> = ({ obj }) => {
   const { t } = useTranslation();
   return (
     <>
```

Comment:
there's an extra fragment wrapper that doesn't need to be there

Reply:
Good catch. Done.